client/marklogicTdeValidateClient.ts

```diff
@@ -127,4 +125,96 @@ export class MarkLogicTdeValidateClient {
             return errorMessage;
         }
     }
+
+    private buildDocumentList(uriVarVal: string): string {
+        const valArray = uriVarVal.split(',');
```

Comment:
This isn't 100% reliable because MarkLogic allows for a URI to have a comma in it. You could come up with another delimiter that isn't allowed in a URI - but I would hold off on this until someone asks for it. I think if someone does want to test multiple existing URIs, the current UX will be fine - i.e. test one, analyze the results; then test another by changing the MLXPRS_TEST_URI value and analyze the results. Or - just write the equivalent of that in an editor and eval it via mlxprs.